config.yaml

```diff
@@ -11,6 +11,18 @@ logging:
   file: ./data/ambianic-log.txt
   level: INFO
 
+# Store notifications provider configuration
+# see https://github.com/caronc/apprise#popular-notification-services for syntax examples
+# notifications:
+#   email:
```

Comment:
If we are going to leave in this flexibility for defining notification targets, can we use as examples more intuitive labels. 
`email` and `alert_fall` do not seem like an intuitive siblings under notifications. Maybe`catch_all_email` and `alert_fall` instead?

Reply:
Ok, done!